swagger.yaml

```diff
@@ -4042,6 +4042,10 @@ definitions:
       ignition_endpoint_token_set:
         type: boolean
         description: True if the token to fetch the ignition from ignition_endpoint_url is set.
+      labels:
```

Comment:
```suggestion
      node_labels:
```

Reply:
We had given it a different name since we are already using node_labels as an array in host-update-params. I observed that to avoid confusion, they have maintained different names in host and host-update-params for role, hostname etc.